src/helper/helper.js

```diff
@@ -263,7 +262,7 @@ function processCommand_(command, model) {
  * If a processor for the command has been registered, the processor function
  * will be invoked with any arguments passed in.
  *
- * @param {*} args The arguments object containing the command
+ * @param {!Object<*>} args The arguments object containing the command
  *     to execute and optional arguments for the processor.
  * @param {!Object<*>|!Array<*>} model The current dataLayer model.
```

Comment:
I think that the model is never an Array<*>, always Object<*>.

Reply:
Great catches, thank you! I've updated them now.